### PR TITLE
store region code and map the code to the relevant region at the frontend.

### PR DESCRIPTION
**Pull request by zhouzilong2020**
Before, we store the region name (e.g. `N. Virginia`) in the persistent file. 
After, a map is used to map the code to the relevant region name in the frontend (e.g. `us-east-1` -->`N. Virginia`).

Also, storing the actual region code is more favorable for the code is needed for API call to the provider, leaving more space for further development. (say, redirect to the actual purchase page in a single click of the instance item).

### CODE REVIEW

**Comment by zhouzilong2020** [comment]
PTAL

File: client/aws/aws.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func (e EngineType) String() string {
 type instance struct {
 	ID             string
 	ServiceCode    string `json:"servicecode"`
```
Comment (d-bytebase) [inline client/aws/aws.go:58]: nit: serviceCode

File: frontend/src/components/CostTable.vue [hunk 2]
```diff
@@ -131,21 +132,13 @@ const columns: any = computed(() => [
...
+          return stringComp;
         }
 
-        // if tow region are identical, sort them with id
-        if (a.length == b.length) {
-          return row1.id - row2.id;
-        }
-
-        return a.length - b.length;
+        // if tow region are identical, sort them with instance's id
```
Comment (d-bytebase) [inline frontend/src/components/CostTable.vue:140]: tow -> two. Use full English sentence for comments, end with period.

// If two regions are identical, sort them with instance id.

**Comment by d-bytebase** [comment]
@LiuJi-Jim Would you help review frontend code? Thanks

File: client/aws/aws.go (hunk 1 above)
Comment (zhouzilong2020) [reply client/aws/aws.go:58]: AWS names this field this way.😮‍💨 I have to adopt their fault.

Comment (d-bytebase) [reply client/aws/aws.go:58]: sad... Add a comment to explain this?

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**